pkgs/by-name/en/envoy-bin/update.sh

```diff
@@ -1,24 +1,24 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl gnused gawk nix-prefetch common-updater-scripts jq
+#!nix-shell -i bash -p common-updater-scripts coreutils gnused ripgrep
```

Comment:
ripgrep missing was causing the bot to fail

https://r.ryantm.com/log/envoy-bin/2025-08-26.log